showcase/src/app/demo-ui/components/table-with-items-per-page-selector/table-with-items-per-page-selector.component.html

```diff
@@ -0,0 +1,14 @@
+<stark-table
```

Comment:
Could you please add an example of this file in the "showcase/src/assets/examples/table/..." ?